msg/Enable.msg

```diff
@@ -0,0 +1 @@
+bool enable
```

Comment:
Is there not a ROS built-in message for this?

Reply:
You can just send primitive types, it would be like std_msgs/bool

Reply:
Sending that would seem reasonable to me? I'm getting kinda confused which message exactly its being used for

Reply:
Ok I guess we can change it... joe u do it first on ur end and then lmk and then ill change it on my end